dds/idl/marshal_generator.cpp

```diff
@@ -2934,10 +2936,7 @@ namespace {
         // ends before some fields on the reader side get their values.
```

Comment:
Can you remove this comment since this PR basically fixes this? (We don't need to involve try construct since the spec just says non-optional fields are set to default.)

Reply:
Done